src/network/networkctl.c

```diff
@@ -2358,6 +2359,9 @@ static int link_status_one(
                         return table_log_add_error(r);
         }
 
+        if (captive_portal)
+                table_add_string_line(table, "Captive Portal:", captive_portal);
```

Comment:
hmm, this should probably escape the field before display, so that embedded control characters/ANSI sequences or so don't fuck things up.

i.e. maybe just call cescape() or so before showing this